prefect_aws/workers/ecs_worker.py

```diff
@@ -215,6 +215,31 @@ def parse_identifier(identifier: str) -> ECSIdentifier:
     return ECSIdentifier(cluster, task)
 
 
+def mask_sensitive_env_values(
+    task_run_request: dict, values: list[str], keep_length=3, replace_with="***"
```

Comment:
i think 3.8 gets mad at list[str] and needs List[str]

Reply:
ahhh good catch, will fix